### ISSUE TITLE
[MAS4.1.2][Screen reader-New bot configuration] Hide and Copy controls role is incorrectly defined 

### ISSUE THREAD

**Issue by Kumar2608** [microsoft/BotFramework-Emulator#1825]
**Actual Result:**
Role is incorrectly defined for Hide and copy controls present in the New bot configuration dialog, Role is defined as link instead of button
Hide is reading as Hide link and copy is reading as Copy link 

**Expected Result:**
Role should be defined as button for Hide and copy controls and NVDA should narrate as Copy button and Hide\Show button 
Note: When the Hide and copy buttons are disabled NVDA is not reading the disabled info.

**Repro Steps:**
Turn on the NVDA
1.	Open the Bot Framework V4 Emulator.
2.	Navigate to the open bot control and select it 
3.	Dialog gest opened
4.	Navigate to the Hide and copy controls 
5.	Observe the issue

**User Impact:**
Users who rely on screen reader will be affected if role is incorrectly defined for controls then the screen reader users will miss guide which type of control.

**Test Environment:**
OS Build: Version 1903(OS Build 18362.295)
Application: Bot Framework V4 Emulator
Screen reader: NVDA 

**MAS Reference Link:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}
[MAS4.1.2_New bot configuration.zip](https://github.com/microsoft/BotFramework-Emulator/files/3574623/MAS4.1.2_New.bot.configuration.zip)

**Comment by corinagum** [comment]
I'm all for fixing these from anchors to buttons. @tonyanziano thoughts?

**Comment by tonyanziano** [comment]
Sounds good to me @corinagum .

Might be a good opportunity to create a `<LinkButton />` component in `sdk/ui-react`

**Comment by Kumar2608** [comment]
Verified in latest bot emulator version 4.5.2-80829 with NVDA(2019.2) issue got fixed now hence we are closing this bug 
Now NVDA is reading as button for Hide and copy controls
